src/main/java/me/zeroeightsix/kami/gui/rgui/component/AbstractComponent.java

```diff
@@ -18,6 +18,8 @@
  */
 public abstract class AbstractComponent implements Component {
 
+    String name;
```

Comment:
why was this added? components already have names, it's inherited from Component